src/Commands/Env/DeployCommand.php

```diff
@@ -15,29 +15,34 @@ class DeployCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Deploy the dev environment to either test or live
+     * Deploys code to the Test or Live environment.
```

Comment:
I don't think the names of environments ought to be capitalized. If the user uses `<site>.Test` or <site>.Live`, Terminus will err.